File: python/sorting algorithm/insertion-sort.py
def insertionSort(array):
    for i in range(1, len(array)):
        j = i
        crnt = array[i]
        while j > 0 and crnt < array[j-1]:
            array[j] = array[j-1]
            j -= 1
        array[j] = crnt
    return arr


arr = [8, 727, 69, 420, 1]
sortedarr = insertionSort(arr)
print(sortedarr)

File: C/sorting algorithm/insertion-sort.c
#include <stdio.h>
int main(void)
{
    int arr[5] = {8, 727, 69, 420, 1};
    int sizeofarr = (int)sizeof(arr) / (int)sizeof(arr[0]);
    for (int i = 1; i < sizeofarr; i++)
    {
        int j = i;
        int crnt = arr[i];
        while (j > 0 && crnt < arr[j - 1])
        {
            arr[j] = arr[j - 1];
            j -= 1;
        }
        arr[j] = crnt;
    }
    for (int i = 0; i < sizeofarr; i++)
    {
        printf("%d\n", arr[i]);
    }
}